coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -407,8 +437,7 @@ def test_dot(self, vector_length, backend):
         vector2 = generate_input_data((vector_length, ))
 
         TorchBaseTest.run_compare_torch((vector1, vector2), model, backend=backend, input_as_shape=False)
-
-
+        
```

Comment:
Another very minor comment. I believe also according to PEP8, you want two blank lines after the end of a class. I know our code often does not follow this convention.

Reply:
its not consistent across the file, but I changed it at the place you asked